ykman/cli/openpgp.py

```diff
@@ -237,6 +248,34 @@ def change_admin(ctx, admin_pin, new_admin_pin):
     controller.change_admin(admin_pin, new_admin_pin)
 
 
+@access.command("set-signature-policy")
+@click.argument("policy", metavar="POLICY", type=EnumChoice(POLICY))
+@click.option("-a", "--admin-pin", help="Admin PIN for OpenPGP.")
+@click.pass_context
+def set_signature_policy(ctx, policy, admin_pin):
+    """
+    Set signature pin policy.
+
+    \b
+    POLICY  Signature pin policy to set (always, once).
+
+    The signature pin policy is used to control whether the pin is
+    always required when using the signature key, or if it is required
+    only once per session.
+    """
+    controller = ctx.obj["controller"]
+
+    if admin_pin is None:
+        admin_pin = click_prompt("Enter Admin PIN", hide_input=True)
+
+    try:
+        controller.verify_admin(admin_pin)
+        controller.set_signature_pin_policy(policy)
+    except Exception as e:
+        logger.debug("Failed to set new signature pin policy", exc_info=e)
+        raise CliFail("Failed to set new signature pin policy")
```

Comment:
Capitalize "PIN". The `logger.debug` line is not needed, the `CliFail` stacktrace will include the exception that caused it automatically.